janusgraph-cassandra/src/main/java/org/janusgraph/diskstorage/cassandra/astyanax/AstyanaxStoreManager.java

```diff
@@ -270,6 +279,7 @@
 
     private final RetryPolicy retryPolicy;
 
+    final int readPageSize;
```

Comment:
this has no modifier on purpose to allow access for the `AstyanaxKeyColumnValueStore` in the same package.